src/index.js

```diff
@@ -51,16 +48,13 @@ function App() {
 
   return (
     <Security {...config} onAuthRequired={authHandler}>
-      {/* hides the Navigation menu when at the login page, since the login page's purpose is singular */}
```

Comment:
This deletion would satisfy the bug fix for https://trello.com/c/h6sjGaeD/60-bugfix-with-new-layout-template-for-pages-navheader-component-is-rendered-twice
and we could close the PR Gabe made for that, right?